continuous-deployment/pipelines/readme.md

```diff
@@ -0,0 +1,17 @@
+# Pipelines
+
+Pipelines is a way to automatically build and test your code project and make it available to other users. It works with just about any language or project type.
+
+Pipelines combines continuous integration (CI) and continuous delivery (CD) to constantly and consistently test and build your code and ship it to any target.
+
+There are a few suggestions listed below to keep in mind when creating pipelines for continuous deployment:
+
+1. Keep your pipelines fast
+1. Isolate and secure your CI/CD environment
+1. Make the CI/CD pipeline the only way to deploy to production
+1. Maintain parity with production wherever possible
+1. Build only once and promote the result through the pipeline
+1. Run your fastest tests early
```

Comment:
So that you can fail out of the build pipeline fast and free up resources for others?

Reply:
Yes, as well as unblock other pipelines to run if limited resources are available. Will update statement to reflect that.